client/src/modules/reports/generate/configurable_analysis_report/configurable_analysis_report.html

```diff
@@ -21,54 +21,64 @@ <h3 class="text-capitalize" translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.TITLE<
 
         <div class="panel-body">
           <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
-            <bh-fiscal-period-select
-              on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscal(fiscal)"
-              on-select-period-from-callback="ReportConfigCtrl.onSelectPeriodFrom(period)"
-              on-select-period-to-callback="ReportConfigCtrl.onSelectPeriodTo(period)">
-            </bh-fiscal-period-select>
 
-            <hr>
-            <bh-multiple-cashbox-select
-              cashbox-ids="ReportConfigCtrl.reportDetails.cashboxes"
-              on-change="ReportConfigCtrl.onSelectCashboxes(cashboxes)"
-              required="true">
-            </bh-multiple-cashbox-select>
-
-            <hr>
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="ReportConfigCtrl.reportDetails.hide_account_details" ng-true-value="0" ng-false-value="1">
-                <span translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.HIDE_ACCOUNT_DETAILS</span>
-              </label>
-            </div>
-            <div class="checkbox">
-              <label>
-                <input type="checkbox" ng-model="ReportConfigCtrl.reportDetails.hide_details_types" ng-true-value="0" ng-false-value="1">
-                <span translate>REPORT.CONFIGURABLE_ANALYSIS_REPORT.HIDE_DETAILS_TYPES</span>
-              </label>
+            <div ng-if="ReportConfigCtrl.configurationsAreMissing" style="margin-bottom: 1em;">
+              <span class="label label-warning text-capitalize" style="padding: 6px;">
+                <i class="fa fa-exclamation-triangle"></i>
+                <span style="font-size: 1.75em;" translate> FORM.INFO.THE_NECESSARY_CONFIG_MISSING </span>
+              </span>
             </div>
 
-            <hr>
-            <bh-yes-no-radios
-              value="ReportConfigCtrl.reportDetails.includeUnpostedValues"
-              on-change-callback="ReportConfigCtrl.onChangeUnpostedValues(value)"
-              name="includeUnpostedValues"
-              label="REPORT.OPTIONS.INCLUDE_UNPOSTED_RECORDS"
-              help-text="REPORT.OPTIONS.INCLUDE_UNPOSTED_RECORDS_HELP">
-            </bh-yes-no-radios>
+            <fieldset
+              ng-disabled="ReportConfigCtrl.configurationsAreMissing"
+              style="{{ReportConfigCtrl.warningStyle}}">
```

Comment:
If you use angularjs templates in styles, you should use `ng-style`.  But either way, I think the regular disabled style is cleaner.  It is correct that this is less obvious about what is going on, but I think the message should be improved, rather than making a bespoke style.

Reply:
Decided to hide all form elements if the configuration is missing.

Reply:
Sounds good.